lms/migrations/versions/efcf8671f4d3_add_the_groups_table.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""Add the groups table."""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = "efcf8671f4d3"
+down_revision = "64d9eacab937"
+
+
+def upgrade():
+    op.create_table(
+        "groups",
```

Comment:
Just an inflection point: Is `groups` definitely the name we want to use for this table? It feels almost more like a (pseudo) relationship (albeit with a remote data source). `groups` might be just fine; I'm musing...